idb/sig_type.go

```diff
@@ -59,5 +59,5 @@ func SignatureType(stxn *transactions.SignedTxn) (SigType, error) {
 		return Lsig, nil
 	}
 
-	return "", fmt.Errorf("unable to determine the signature type")
+	return "", fmt.Errorf("unable to determine the signature type for stxn=%+v", stxn)
```

Comment:
@winder - should I remove this very verbose addition? This is what I added following our [recent slack thread](https://algorand-internal.slack.com/archives/C028R5W4DA6/p1656452142169899?thread_ts=1656419400.435169&cid=C028R5W4DA6). But maybe it's actually useful to keep for the rare occasion that this would occur.